proposals/binary-compat-only.md

```diff
@@ -0,0 +1,123 @@
+# BinaryCompatOnlyAttribute
+
+## Summary
+[summary]: #summary
+
+We introduce a new attribute, `System.BinaryCompatOnlyAttribute`, that causes the type or member to which it is applied to be entirely inaccessible from source code.
+It will still be emitted as declared, with the accessibility declared, but will be treated as if it does not exist by all other compile-time C# mechanisms.
```

Comment:
so can a method not call itself?

what if i have multiple BCOA members, where one calls the other.  i feel like that would be totally normal.